### PR TITLE
Fixes to add in missing CapabilityStatement documentation

### PR DESCRIPTION
**Pull request by dt-r**
This PR includes the following:
- updates the narrative html to include the missing multipleAnd & multipleOr requirements that are visible on the Profile Notes pages but not in the narrative of capability statements
- updates CapabilityStatement.rest.security.description to match what is in the narrative
- updates the documentation element for search parameters in the CapabilityStatements to include multipleAnd & multipleOr requirements

### CODE REVIEW

File: input/resources/au-core-requester.xml [hunk 1]
```diff
@@ -970,6 +976,8 @@
     <div>
       <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
       <p>The responder <strong>SHALL</strong> support both.</p>
+      <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:979]: This should be **SHALL** 
See AU Core Encounter notes:
SHALL support _[multipleOr](http://hl7.org/fhir/R4/searchparameter-definitions.html#SearchParameter.multipleOr)_ search on status (e.g.status={system|}[code],{system|}[code],...)

File: input/resources/au-core-requester.xml [hunk 2]
```diff
@@ -970,6 +976,8 @@
...
+      <p>The responder <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:980]: This should be **SHALL**.

File: input/resources/au-core-responder.xml [hunk 3]
```diff
@@ -986,6 +988,8 @@
     <div>
       <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
       <p>The responder <strong>SHALL</strong> support both.</p>
+      <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:991]: This should be SHALL for both requester & responder.

File: input/resources/au-core-responder.xml [hunk 4]
```diff
@@ -3273,7 +3293,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/DocumentReference-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:3296]: The expectation for 'multipleOr' should be **SHALL**

File: input/resources/au-core-requester.xml [hunk 5]
```diff
@@ -3254,7 +3280,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/DocumentReference-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:3283]: The expectation in 'multipleOr' should be **SHALL**

File: input/resources/au-core-requester.xml [hunk 6]
```diff
@@ -1609,6 +1621,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:1624]: Should be SHALL for both responder & requester.

File: input/resources/au-core-requester.xml [hunk 7]
```diff
@@ -3704,7 +3730,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/medications-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:3733]: The expectation for 'multipleOr' should be **SHALL**.

File: input/resources/au-core-requester.xml [hunk 8]
```diff
@@ -2720,6 +2740,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
+    <p>The responder <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:2744]: Should be SHALL for both responder & requester.

File: input/resources/au-core-responder.xml [hunk 9]
```diff
@@ -2778,6 +2796,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
+    <p>The responder <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:2800]: This should be SHALL for both requester & responder.

File: input/resources/au-core-responder.xml [hunk 10]
```diff
@@ -1628,6 +1636,8 @@
...
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:1640]: This should be SHALL for both requester & responder.

File: input/resources/au-core-requester.xml [hunk 11]
```diff
@@ -4522,7 +4548,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/Procedure-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:4551]: The expectation for 'multipleOr' should be SHALL.

File: input/resources/au-core-responder.xml [hunk 12]
```diff
@@ -4533,7 +4553,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/Procedure-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:4556]: The expectation for 'multipleOr' should be SHALL.

File: input/resources/au-core-requester.xml [hunk 13]
```diff
@@ -1850,6 +1868,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
+    <p>The responder <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:1872]: Should be SHALL for both responder & requester.

File: input/resources/au-core-responder.xml [hunk 14]
```diff
@@ -1871,6 +1885,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
+    <p>The responder <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:1889]: Should be SHALL for both responder & requester.

File: input/resources/au-core-responder.xml [hunk 15]
```diff
@@ -3939,7 +3959,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/Observation-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:3962]: The expectation for 'multipleOr' should be SHALL

File: input/resources/au-core-requester.xml [hunk 16]
```diff
@@ -3924,7 +3950,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/Observation-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:3953]: The expectation for 'multipleOr' should be SHALL

File: input/resources/au-core-requester.xml [hunk 17]
```diff
@@ -1850,6 +1868,8 @@
   <div>
     <p>The requester <strong>SHALL</strong> provide at least a code value and <strong>MAY</strong> provide both the system and code values.</p>
     <p>The responder <strong>SHALL</strong> support both.</p>
+    <p>The requester <strong>SHOULD</strong> support <code>multipleOr</code>.</p>
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:1871]: This should be SHALL.

File: input/resources/au-core-requester.xml [hunk 18]
```diff
@@ -2756,6 +2780,8 @@
...
```
Comment (dbojicic) [inline input/resources/au-core-requester.xml:2783]: Should be SHALL for both responder & requester.

File: input/resources/au-core-requester.xml (hunk 16 above)
Comment (dbojicic) [inline input/resources/au-core-requester.xml:3953]: The expectation for 'multipleOr' should be SHALL.

File: input/resources/au-core-responder.xml [hunk 19]
```diff
@@ -3719,7 +3739,7 @@
 <name value="status"/>
 <definition value="http://hl7.org/fhir/SearchParameter/medications-status"/>
 <type value="token"/>
-<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both."/>
+<documentation value="The requester **SHALL** provide at least a code value and **MAY** provide both the system and code values.&#xA;&#xA;The responder **SHALL** support both.&#xa;&#xa;The requester **SHOULD** support `multipleOr`.&#xa;&#xa;The responder **SHOULD** support `multipleOr`."/>
```
Comment (dbojicic) [inline input/resources/au-core-responder.xml:3742]: The expectation for 'multipleOr' should be SHALL.

File: input/resources/au-core-responder.xml (hunk 15 above)
Comment (dbojicic) [inline input/resources/au-core-responder.xml:3962]: The expectation for 'multipleOr' should be SHALL

File: input/resources/au-core-requester.xml (hunk 1 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 2 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 3 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 15 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 19 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 16 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 18 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 17 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 16 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 15 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 14 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 13 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 12 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 11 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 10 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 9 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 8 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 7 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 6 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 5 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 4 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-responder.xml (hunk 3 above)
Comment (dt-r) [reply input/resources/au-core-responder.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 2 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

File: input/resources/au-core-requester.xml (hunk 1 above)
Comment (dt-r) [reply input/resources/au-core-requester.xml]: fixed

**Review by dbojicic — APPROVED** [review]

**Pull request merged**